src/flask_jsonrpc/site.py

```diff
@@ -92,7 +92,7 @@ def is_json(self) -> bool:
         https://github.com/pallets/werkzeug/blob/master/src/werkzeug/wrappers/json.py#L54
         """
         mt = request.mimetype
-        return mt == 'application/json' or mt.startswith('application/') and mt.endswith('+json')
+        return mt == 'application/json' or mt == 'application/json-rpc' or mt.startswith('application/') and mt.endswith('+json')
```

Comment:
It's not feasible to do that, I think put this on set up configuration instant of here.

Reply:
See https://github.com/cenobites/flask-jsonrpc/issues/222#issuecomment-1124989201.

Reply:
Could you kindly explain why it's not feasible? If I just make this little change and use it before the new version is released, would there be any risks? I briefly tested it, and it seems to be functioning properly.

Reply:
Looking forward to your reply. I need this package badly. Thank you.

Reply:
Because it is specific for your use case, and in JSON-RPC specification[1] there isn't a comment about that, for that reason I prefer to put it on set up configuration instead of in core code. For example, tomorrow another user there is a new Content-Type, so it isn't feasible to put this on conditions every time that need to.

[1] - https://www.jsonrpc.org/specification

Reply:
@hualongsd I found out this Content-Type in specification 1.2[1] (JSON-RPC over HTTP), I will accept this PR if some changes ;).


[1] - https://www.jsonrpc.org/historical/json-rpc-over-http.html#id13